### PR TITLE
BAK-1141: Update initial data load for development

### PR DESCRIPTION
**Pull request by THOR300**
Integrated the sectors data from the recent data dump as an example prior to completing all required data sources. 

These are now accessible through the UI as part of the filters.

### CODE REVIEW

File: backend/app/data_migrations/populate_sector.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import csv
+
+from sqlalchemy.orm import Session
+
+from app.db.models import Sector
+from .utils import has_rows
+
+
+def populate_sector(db: Session) -> None:
```
Comment (joel-wright) [inline backend/app/data_migrations/populate_sector.py:9]: this is exactly the sort of thing we need, but let's talk about making the sector data independent of pre-existing id/parent_id fields.

**Review by joel-wright — COMMENTED** [review]
Nice start, shall we have a chat later today?

**Comment by joel-wright** [comment]
Created dump files for all metadata tables, unified format & loading

File: backend/app/data_migrations/utils.py [hunk 2]
```diff
@@ -1,7 +1,54 @@
...
+
+        child_nodes = cast(Sequence[Mapping[str, Mapping]], entry["children"])
+        if child_nodes:
+            db.flush()
+            _load_tree(db, table, child_nodes, parent_db_entry.id)
+
+
+def load_list(db: Session, table: Base, data_list: Sequence[Mapping]) -> None:
+    """
+    Load a list od data stored as JSON into a database table
```
Comment (THOR300) [inline backend/app/data_migrations/utils.py:47]: Typo here "List of data..."

File: backend/app/data_migrations/populate_topic.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+from sqlalchemy.orm import Session
+
+from app.db.models import Response
+from .utils import has_rows, load_list
+
+
+# TODO: Rename Response -> Topic
+def populate_response(db: Session) -> None:
+    """Populate responses from CSV file."""
```
Comment (THOR300) [inline backend/app/data_migrations/populate_topic.py:11]: The comment here references a csv where we are using json (same as some other functions)

**Review by THOR300 — COMMENTED** [review]
Looks really good just a couple of comments that need updating. 

Ran locally and verified in beekeeper that the data loaded as well. 

Do we need more extensive tests to verify that the metadata has been loaded?

File: backend/app/initial_data.py [hunk 4]
```diff
@@ -26,6 +27,7 @@ def run_data_migrations(db):
     populate_language(db)
     populate_document_type(db)
     populate_geography(db)
+    populate_sector(db)
```
Comment (diversemix) [inline backend/app/initial_data.py:30]: We probably need to add the population of meta data here :)

File: backend/.pre-commit-config.yaml [hunk 5]
```diff
@@ -8,6 +8,7 @@ repos:
     rev: 4.0.0
     hooks:
       - id: flake8
+        args: ["--per-file-ignores", "__init__.py:F401"]
```
Comment (diversemix) [inline backend/.pre-commit-config.yaml:11]: nice :+1:

**Review by diversemix — APPROVED** [review]
LGTM

**Pull request merged**